Control coverage/Feature/AppService.md

```diff
@@ -17,6 +17,8 @@
 - [Azure_AppService_AuthZ_Configure_IP_Restrictions](#azure_appservice_authz_configure_ip_restrictions)
 - [Azure_AppService_AuthN_Use_Managed_Service_Identity](#azure_appservice_authn_use_managed_service_identity)
 
+- [Azure_AppService_DP_Dont_Use_FTP_Deployment](#azure_appService_dp_dont_use_ftp_deployment)
```

Comment:
@pranchalsomani The link doesn't seem to work, Can you please check?

Reply:
Link is working now.